### PR TITLE
docs: discontinued admob

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

revert documentation link to deprecated admob. 

## Related Issues
 none.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: website/plugins.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 module.exports = [
   {
-    name: 'AdMob',
-    pub: 'google_mobile_ads',
+    name: 'AdMob (Deprecated)',
+    pub: 'firebase_admob',
     firebase: 'admob',
-    remoteSource: 'https://github.com/googleads/googleads-mobile-flutter',
-    documentation: 'https://pub.dev/documentation/google_mobile_ads/latest/',
+    documentation: '',
```
Comment (rrousselGit) [inline website/plugins.js:6]: Shouldn't this link to    ` documentation:'https://firebase.flutter.dev/docs/analytics/overview'`, ?

Comment (russellwheatley) [reply website/plugins.js:6]: An empty string renders no link on the UI like it was previously:
Screenshot 2021-03-25 at 16 52 22

File: website/src/pages/index.tsx [hunk 2]
```diff
@@ -78,6 +78,10 @@ function Home() {
         </div>
       </section>
       <main>
+          {/* Temporary. remove note when admob fully deprecated */}
+          <div style={{display:'flex', justifyContent:'center', paddingTop:'10px'}}>
+            The AdMob package is deprecated and will be removed in the future. We recommend using the	&nbsp;<a href="https://pub.dev/packages/google_mobile_ads" target="_blank">google_mobile_ads</a>	&nbsp;package.
```
Comment (Salakar) [inline website/src/pages/index.tsx:83]:
```suggestion
            The FlutterFire AdMob package is deprecated and will be removed in the future. We recommend switching to the official <a href="https://pub.dev/packages/google_mobile_ads" target="_blank">google_mobile_ads</a> package by Google. A migration guide is available <a href="https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_admob#deprecation-of-firebase_admob-plugin" target="_blank">here</a>.
```

File: website/src/pages/index.tsx [hunk 3]
```diff
@@ -78,6 +78,10 @@ function Home() {
         </div>
       </section>
       <main>
+          {/* Temporary. remove note when admob fully deprecated */}
+          <div style={{display:'flex', justifyContent:'center', paddingTop:'10px'}}>
+            The FlutterFire AdMob package is deprecated and will be removed in the future. We recommend switching to the official <a href="https://pub.dev/packages/google_mobile_ads" target="_blank">google_mobile_ads</a> package by Google. A migration guide is available <a href="https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_admob#deprecation-of-firebase_admob-plugin" target="_blank">here</a>.
```
Comment (Salakar) [inline website/src/pages/index.tsx:83]:
```suggestion
            The FlutterFire AdMob package is deprecated and will be removed in the future. We recommend switching to the official &nbsp;<a href="https://pub.dev/packages/google_mobile_ads" target="_blank">google_mobile_ads</a>&nbsp; package by Google. A migration guide is available &nbsp;<a href="https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_admob#deprecation-of-firebase_admob-plugin" target="_blank">here</a>.
```

**Pull request merged**